### PR TITLE
MINOR: Fix typos for connect

### PR DESCRIPTION
**Pull request by berg223**
*More detailed description of your change,
Fix typos for connect*

### Committer Checklist (excluded from commit message)
- [x] Verify design and implementation 
- [x] Verify test coverage and CI build status
- [x] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java [hunk 1]
```diff
@@ -271,12 +271,12 @@ public synchronized void shutdown() {
 
     void doShutdown() {
         try {
-            TargetState preEmptedState = pendingTargetStateChange.getAndSet(null);
+            TargetState preEmptiedState = pendingTargetStateChange.getAndSet(null);
```
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: This change is incorrect; "pre-empted" is the intended term.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java [hunk 2]
```diff
@@ -325,20 +325,20 @@ public boolean awaitShutdown(long timeoutMs) {
     }
 
     public void transitionTo(TargetState targetState, Callback<TargetState> stateChangeCallback) {
-        Callback<TargetState> preEmptedStateChangeCallback;
-        TargetState preEmptedState;
+        Callback<TargetState> preEmptiedStateChangeCallback;
```
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Also incorrect

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java [hunk 3]
```diff
@@ -325,20 +325,20 @@ public boolean awaitShutdown(long timeoutMs) {
...
+        TargetState preEmptiedState;
```
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Also incorrect

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java [hunk 4]
```diff
@@ -325,20 +325,20 @@ public boolean awaitShutdown(long timeoutMs) {
...
-            preEmptedStateChangeCallback.onCompletion(
+        if (preEmptiedStateChangeCallback != null) {
+            preEmptiedStateChangeCallback.onCompletion(
                     new ConnectException(
-                            "Could not begin changing connector state to " + preEmptedState.name()
+                            "Could not begin changing connector state to " + preEmptiedState.name()
                                 + " before another request to change state was made;"
                                 + " the new request (which is to change the state to " + targetState.name()
-                                + ") has pre-empted this one"),
+                                + ") has pre-emptied this one"),
```
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Also incorrect

**Review by C0urante — COMMENTED** [review]
Thanks @xiaocairush! LGTM except for the "pre-empted" changes. Once those are removed we can merge.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java (hunk 3 above)
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: resolved

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java (hunk 4 above)
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: resolved

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java (hunk 1 above)
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Have reverted all false positive fix in WorkerConnector

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java (hunk 2 above)
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: resolved

**Comment by C0urante** [comment]
Thanks! Will merge pending CI build.

**Review by C0urante — APPROVED** [review]

**Pull request merged**